packages/components/package.json

```diff
@@ -40,7 +40,7 @@
     "typescript": "^4.8.3"
   },
   "dependencies": {
-    "@jpmorganchase/mosaic-store": "^0.1.0-beta.36",
```

Comment:
NOTE: Removed shady dependency on store by moving ThemeProvider to site-components